### PR TITLE
Chore: remove unused files from .gitignore

### PR DESCRIPTION
**Pull request by not-an-aardvark**
**What is the purpose of this pull request? (put an "X" next to item)**

[x] Other, please explain:

**What changes did you make? (Give an overview)**

The `test.js` file has been in gitignore since the initial commit of this repo. I'm not sure why it was ignored initially, but I'm pretty sure that name doesn't need to be ignored anymore.

The `.eslintcache` file doesn't need to be ignored because we use `.cache/js_cache` rather than `.eslintcache` for our cache file.

**Is there anything you'd like reviewers to focus on?**

Nothing in particular

### CODE REVIEW

**Comment by gyandeeps** [comment]
I use test.js file in many projects just to run one of stuff... Since it doesn't hurt, I would say don't remove it... I think someone else also uses this... Not sure who...

File: .gitignore [hunk 1]
```diff
@@ -9,7 +8,6 @@ tmp/
 jsdoc/
 versions.json
 *.iml
-.eslintcache
```
Comment (alberto) [inline .gitignore]: It looks like this is used by atom's feeling package

**Comment by mysticatea** [comment]
I'm using `test.js`, too.

File: .gitignore (hunk 1 above)
Comment (alberto) [inline .gitignore]: Feeling was supposed to read eslint 😂

**Comment by not-an-aardvark** [comment]
Ok, seems like it's not worth removing these, so I'll close this PR.

**Pull request closed without merging**